aimsprop/plot.py

```diff
@@ -189,7 +189,7 @@ def plot_population(
     bundles: list,
     time_units: str = "au",
     legend_loc="right",
-    state_colors: list = None,
+    state_colors: list = [],
```

Comment:
We actually want to leave this as `= None`. A surprising thing in python is that if you declare a container (like a list or dictionary) in the function signature it will become a singleton across the entire application--meaning that each time you call the function the same array is used under the hood. This can lead to some unexpected behavior! So if you want to declare a blank array in a function you'd do this following:

```
def my_func(container: Optional[List[int]] = None):
    container = container or []
    ...
```